crates/re_arrow_store/src/store_gc.rs

```diff
@@ -13,6 +16,31 @@ pub enum GarbageCollectionTarget {
     ///
     /// The fraction must be a float in the range [0.0 : 1.0].
     DropAtLeastFraction(f64),
+
+    /// GC Everything that isn't protected
+    Everything,
```

Comment:
Two differences:
 - DropAtLeastFraction does size book-keeping and potentially decides to stop early. I was running into edge cases where it would conclude it had met the 1.0 threshold, but in fact still had stuff it could GC.
 - DropAtLeastFraction needs to do an oldest-first incremental traversal. Everything has lots of room for optimizations.
 
Special-casing the numerical value of 1.0 felt weirder to me than special-casing on an explicit enum but I can make that switch if you think it's clearer.

Reply:
No, that makes sense :+1: